doc/man/8/ceph-fuse.rst

```diff
@@ -16,10 +16,10 @@ Description
 ===========
 
 **ceph-fuse** is a FUSE ("Filesystem in USErspace") client for Ceph
-distributed file system. It will mount a ceph file system specified
-via the -m option or described by ceph.conf (see below) at the
-specific mount point. See `Mount CephFS using FUSE`_ for detailed
-information.
+distributed file system. It will mount a ceph file system specified via the -m
+option or described by ceph.conf (see below) at the specific mount point. By
+default, CephX user admin is used to mount CephFS. See `Mount CephFS using
```

Comment:
As I noted in the your other doc PR, let's not mention the admin key in docs.

Reply:
Removed the "admin" bit here.

Reply:
Although, I must ask why? I think at least man page should mention it.

Reply:
Agreed. If you don't specify a -n option then it defaults to "admin". The user ought to be informed of that default.

Reply:
It's arguably not a good default which is why I don't want to draw attention to it in the docs. Abuse of the admin key creates problems. Mentioning it as a default encourages users to deploy the admin key everywhere.